js/lib/ilib.js

```diff
@@ -129,7 +129,7 @@ ilib._getPlatform = function () {
     		}
     	} catch (e) {}
     	
-        if (typeof(process) !== 'undefined' && typeof(window) === 'undefined' && typeof(module) !== 'undefined') {
```

Comment:
Is there specific reason to rollback as previous code? This changes from Enact (Enyo+React) teams's report.
fyi.
https://github.com/enyojs/iLib/pull/94